.github/workflows/on-push-lint-charts.yml

```diff
@@ -0,0 +1,67 @@
+name: Lint and Test Charts
+
+on:
+  push:
+    paths:
+      - 'charts/**'
+      - '.github/**'
+env:
+  KUBE_SCORE_VERSION: 1.10.1
+  HELM_VERSION: 3.5.2
+  CHECK_VERSION_INCREMENT: false # avoid tests failing when multiple attempts to push chart changes are made
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v${{ env.HELM_VERSION }}
+
+      - name: Set up kube-score
+        run: |
+          wget https://github.com/zegl/kube-score/releases/download/v${{ env.KUBE_SCORE_VERSION }}/kube-score_${{ env.KUBE_SCORE_VERSION }}_linux_amd64 -O kube-score
```

Comment:
I'm not quite sure about the value here since we ignore most of the stuff anyway + we only check the rendered chart and not the other yaml files.

Reply:
We're still subject to all the rest of the tests at https://github.com/zegl/kube-score/blob/master/README_CHECKS.md - the value (for me at least), is that we raise the bar on the chart for users (like me) who require kube-score passing in order to deploy into their cluster via CI. It'll also catch any snafus in future PRs, like accidentally turning on privileged mode in the deployment.